lib/generate-code.js

```diff
@@ -21,7 +24,7 @@ module.exports = function generateCode(
   }
 
   output +=
-    `var __options__ = typeof exports.default === 'function' ` +
+    `var ${namespace} = typeof exports.default === 'function' ` +
```

Comment:
Any specific reason to move the constant to another file? It seems to be used elsewhere when it wasn't before - why is that?

Reply:
Because it was being hardcoded inside of vue-i18n-jest and we refactored this, which broke vue-i18n-jest if we were to have released v4 of vue-jest... We should make any required constants passed in rather than requiring developers to dig into vue-jest source code to figure out what constants we're using

Reply:
Right, makes sense, sounds good